apollo-training/src/config/configurations.js

```diff
@@ -10,6 +10,9 @@ const envVarsSchema = Joi.object({
     .default('development'),
   PORT: Joi.number()
     .default(9001),
+  SERVICE_URL: Joi.string()
```

Comment:
What is JOI

Reply:
Joi is a validator package, we create a schema for field and by joi we can validate request body. 
like Email: Joi.email().ToLowercase().required() and 
      Password: Joi.min(8).max(20).required()   etc